### PR TITLE
Changed css of browse button to match the general style of the form

### LINKED ISSUE #210: Override /users/username/profile/ choose file button to the general style guide of the website
For registration/authentication we use django-allauth. This package has basic templates for the required pages. Almost all of them have already been overriden. Override the choose file button to the general style guide of the website. If you find something else feel free to update that too.

### PR DESCRIPTION
**Pull request by abhi20161997**
### Description
Changed css of browse button to match the general style of the form buttons like the cancel button.

Fixes #210 

### Mocks
Before :
new

After :
screenshot from 2018-06-17 00-21-24

### Type of Change:
- User Interface

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by abhi20161997** [comment]
@prachi1210 @monajalal @yatna @sa7i . Please can you review this ?

**Comment by monajalal** [comment]
Can you please add a little detail in front of each bullet item? Like how was the change. One line or so for each suffices. Thanks

**Comment by abhi20161997** [comment]
@monajalal , Can u please elaborate ? I didn't get what u intend to say.

File: systers_portal/templates/community/edit_profile.html [hunk 1]
```diff
@@ -22,4 +22,4 @@ <h1>Edit {{ community }} profile</h1>
 
 {% block community_footer %}
   {% include 'community/snippets/footer.html' %}
-{% endblock %}
+{% endblock %}
```
Comment (sa7i) [inline systers_portal/templates/community/edit_profile.html]: You could remove this change.

File: systers_portal/templates/base.html [hunk 2]
```diff
@@ -165,6 +165,7 @@
 
 <script src="{% static 'js/libs/jquery-1.11.1.min.js' %}"></script>
 <script src="{% static 'js/libs/bootstrap.min.js' %}"></script>
+<script src="{% static 'js/edit_profile.js' %}"></script>
```
Comment (sa7i) [inline systers_portal/templates/base.html]: The file name can be renamed to `profile_choose_file.js` or a better name letting the developers know that the file is for the Choose file button in the edit profile form.

File: systers_portal/static/js/edit_profile.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+$("#upload_pic").click(function() {
+	$("#id_profile_picture").click();
+});
+
+document.getElementById("id_profile_picture").onchange = function () {
+  var str = this.value;
+  var pathTokens = str.split("\\");
+  $("#file_name_display").html(" " + pathTokens[pathTokens.length - 1]);
+};
```
Comment (sa7i) [inline systers_portal/static/js/edit_profile.js]: You can add comments in this file about
1. What this file does and
2. What each component does
You can add a new line at the end of the file.

File: systers_portal/static/js/edit_profile.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+var htmlInnerContent="<label for='id_profile_picture' \n" +
```
Comment (sa7i) [inline systers_portal/static/js/edit_profile.js]: The HTML in the lines 1-7  can be made pretty I think.

File: systers_portal/templates/community/edit_profile.html (hunk 1 above)
Comment (abhi20161997) [reply systers_portal/templates/community/edit_profile.html]: I tried undoing it but it still reflects in the PR. There is no difference between the statements though.

**Comment by abhi20161997** [comment]
@yatna @sa7i  I have applied your suggested changes. @sa7i  Regarding the prettifying, as it was an extremely long statement and I had to break it to avoid style errors I had to follow the format.

**Comment by abhi20161997** [comment]
@monajalal Those bullets are for mentioning the type of change and come under the default structure of a PR. I think adding descriptions there could mess with the default structure.

File: systers_portal/templates/community/edit_profile.html (hunk 1 above)
Comment (sa7i) [reply systers_portal/templates/community/edit_profile.html]: Does it still exist if you add a new line after `{% endblock %}`?

**Comment by abhi20161997** [comment]
@sa7i . Thanks for suggesting the newline. It doesn't show now.

File: systers_portal/static/js/choose_profile_pic.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+// When custom button is clicked, perform a click on file upload field
+$("#upload_pic").click(function() {
+	$("#id_profile_picture").click();
+});
+
+document.getElementById("id_profile_picture").onchange = function () {
+  var str = this.value;
+  var pathTokens = str.split("\\");
+  $("#file_name_display").html(" " + pathTokens[pathTokens.length - 1]);
+};
```
Comment (sa7i) [inline systers_portal/static/js/choose_profile_pic.js]: Please add a new line here.

**Comment by abhi20161997** [comment]
@sa7i, done adding the newline.

File: systers_portal/static/css/style.css [hunk 6]
```diff
@@ -150,6 +150,13 @@ body > .container, body > .container-fluid {
   border-color: #005c88;
 }
 
+.btn-default {
+  background-color: #e2e2e2;
+  border-color: #dadada;
+  color: #555555;
+  padding: 8px 12px 7px;
```
[the remark this answers is not in the document]
Comment (abhi20161997) [reply systers_portal/static/css/style.css]: Basically these are the paddings used for the rest of the similar buttons, therefore I used the same to follow the pattern and maintain uniformity.

**Pull request merged**